manual/manual/cmds/top.etex

```diff
@@ -126,6 +126,11 @@ The following command-line options are recognized by the "ocaml" command.
 \begin{unix}
 The following environment variables are also consulted:
 \begin{options}
+\item["OCAMLTOP_INCLUDE_PATH"] Additional directories to search for compiled
+  object code files (".cmi", ".cmo" and ".cma"). The specified directories are
+  considered from left to right, after the include directories specified on the
+  command line via "-I" have been searched.
```

Comment:
@nojb a  "Since 4.08" would also be nice here.

Reply:
Done.